src/course-outline/card-header/CardHeader.tsx

```diff
@@ -175,19 +176,11 @@ const CardHeader = ({
         ) : (
           <>
             {titleComponent}
-            {readyToSync && (
-              <IconButton
-                className="item-card-button-icon"
-                data-testid={`${namePrefix}-sync-button`}
-                alt={intl.formatMessage(messages.readyToSyncButtonAlt)}
-                iconAs={SyncIcon}
-                onClick={onClickSync}
-              />
-            )}
-            <IconButton
+            <IconButtonWithTooltip
               className="item-card-button-icon"
               data-testid={`${namePrefix}-edit-button`}
               alt={intl.formatMessage(messages.altButtonEdit)}
+              tooltipContent={<div>{intl.formatMessage(messages.altButtonEdit)}</div>}
```

Comment:
This tooltip says "Edit" but when I click on it, it renames the unit. I think it should say "Rename".

<img width="991" height="202" alt="Screenshot 2025-08-27 at 1 51 07 PM" src="https://github.com/user-attachments/assets/3d470cc5-a948-43de-947d-dd69b8c15518" />

<img width="926" height="202" alt="Screenshot 2025-08-27 at 1 51 39 PM" src="https://github.com/user-attachments/assets/6313f0ac-7c2f-41a8-8e28-d96e3adf64d6" />

Reply:
It's true. Updated: https://github.com/openedx/frontend-app-authoring/pull/2319/commits/d4b6e192d1f9b2928f848a85a5f9c50a48c5198f